python/pyproject.toml

```diff
@@ -105,6 +105,7 @@ test = [
   "matplotlib",
   "pandas",
   "peft",
+  "polars",
```

Comment:
why do we need to add `polars` to the dependency?
it looks like `python/sglang/srt/debug_utils/text_comparator.py` has not been used or imported

Reply:
gotcha